fossevents/templates/base.html

```diff
@@ -16,14 +16,39 @@
     {% compress css %}
     <link rel="stylesheet" type="text/css" href="{% static 'css/normalize.css' %}" />
     <link rel="stylesheet" type="text/x-scss" href="{% static 'css/main.scss' %}" />
+
+      <link type="text/css" href="{% static 'css/bootstrap.min.css' %}" rel="stylesheet">
```

Comment:
After seeing the effect of bootstrap framework on the html markups and the css written in this PR. Which is totally non-readable. I can't allow bootstrap to be used.  Please do not use it.